Diagnostic/diagnostic.py

```diff
@@ -450,7 +450,7 @@ def start_daemon():
     raise an exception (often OSError)
     :return: None
     """
-    args = ['python', g_diagnostic_py_filepath, "-daemon"]
+    args = ['python2', g_diagnostic_py_filepath, "-daemon"]
```

Comment:
Any chance that a Python2 distribution _doesn't_ have a 'python2' command?

Reply:
we use the same logic for AMA/OMS and for both of those we check using that, and haven't seen any issue. so i think it should be fine